.github/workflows/lint.yml

```diff
@@ -8,11 +8,13 @@ on:
 
 jobs:
   lint:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
@acevif 

Is this needed to use swiftlint 0.46.5?

Reply:
@acevif 

I think it would be okay to just use `ubuntu-latest` for now and deal with it when it breaks.

`ubuntu-20.04` is also okay, but it would be better to add the following comment.

```yaml
// Follow https://github.com/Cyberbeni/install-swift-tool/blob/master/.github/workflows/test.yml#L9
runs-on: ubuntu-20.04
```

Reply:
@mono0926 Thanks!

I tried applying the latter option.

How is it?